blog/templates/blog/view_entry.html

```diff
@@ -0,0 +1,8 @@
+{% extends "blog/blogbase.html" %}
+
+{% block blog_content %}
+    <h2>{{ entry.title }}</h2>
+    <p>{{ entry.body }}</p>
+    <p>{{ entry.created_dttm }}</p>
```

Comment:
look at the django date filter for how to format this date in a nicer format:

- https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#date